ayan-biswas0412.md

```diff
@@ -0,0 +1,49 @@
+---
+name: "@ayan-biswas0412"
+project: "Camera Based Door Access Control System"
+---
+
+# Camera Based Door Access Control System
+
+## Summary
+
+I am going to build a camera based door access control system that uses facial recognition to grant or deny access to a locked door. 
+The system will include a camera mounted near the door, a small computer for processing the video feed and running the facial recognition algorithm, and a lock mechanism that can be controlled by the computer. 
+I am excited to build this because I think it has the potential to improve security and convenience in a variety of settings, such as office buildings, apartment complexes, and homes.
+
+## Plan
+
+1. Gather materials and tools needed:
+    * Raspberry Pi microcomputer
+    * Raspberry Pi camera module
+    * Servo motor
+    * Lock mechanism (such as a deadbolt or latch bolt)
+    * Wires and connectors
+    * Power supply
+2. Set up the Raspberry Pi and camera module according to the manufacturer's instructions.
+3. Connect the servo motor to the Raspberry Pi and wire it to control the lock mechanism.
```

Comment:
do you plan on breadboarding this or soldering the components together? and, do you have the tools you need?

Reply:
I plan to solder them together, and yes I have necessary soldering iron and other stuffs in my home.

Reply:
great, thank you!